docker/entrypoint-istgtimage.sh

```diff
@@ -20,6 +20,7 @@ touch /usr/local/etc/istgt/logfile
 export externalIP=0.0.0.0
 service rsyslog start
 
-export LD_PRELOAD=/usr/lib/x86_64-linux-gnu/libjemalloc.so
+ARCH=$(uname -m)
```

Comment:
unused? What's the purpose of this variable?

Reply:
I used to define a OS=$(uname) variable which is unused. This one($ARCH) is used to define the path of LD_PRELOAD. export LD_PRELOAD=/usr/lib/${ARCH}-linux-gnu/libjemalloc.so

Reply:
I missed that, thanks. Looks good.